lua/default-config.lua

```diff
@@ -223,7 +223,7 @@ O = {
       -- @usage can be clippy
       formatter = {
         exe = "rustfmt",
-        args = { "--emit=stdout" },
+        args = { "--emit=stdout --edition=2018" },
```

Comment:
can you make it into an actual array like
```lua
args = { "--emit=stdout", "--edition=2018" },
```

Reply:
Done.